.github/workflows/testing.yml

```diff
@@ -0,0 +1,31 @@
+name: Go Test
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.16
```

Comment:
You should also include the "tip" version, as it was before (the latest Go version)

Reply:
Looks good but may lead to bloat: (Linux, Windows, Mac)×(1.16,tip)
Splitting to different parallel workflows may help, will try this

Reply:
I think you can use matrix builds for that. Here is an example how I did it with my library and I think it works well: https://github.com/grongor/panicwatch/blob/master/.github/workflows/ci.yaml

Reply:
Thanks, didn'k know about matrix feature in Actions